### PR TITLE
Fix reload endpoint and Added docs to use this endpoint

### LINKED ISSUE #199: Document the config's reloader endpoint
Document the feature implemented here. https://github.com/python-microservices/pyms/issues/71

### PR DESCRIPTION
**Pull request by avara1986**
Fixes #199

Changes proposed in this PR:
- Added documentation to use the reload endpoint
- Fix error to get default path to reload

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -167,3 +167,29 @@ spec:
 ```
 
 See [Routing](routing.md) and [Examples](examples.md) to continue with this tutorial
+
+## Reload configuration without stop your services
```
Comment (alexppg) [inline docs/configuration.md:171]:
```suggestion
## Reload configuration without stopping your services
```

File: docs/configuration.md [hunk 2]
```diff
@@ -167,3 +167,29 @@ spec:
...
+
+In production environment, you could need to change configuration files without restart the microservice/pod/container.
```
Comment (alexppg) [inline docs/configuration.md:173]:
```suggestion
In a production environment you may need to change the microservice's configuration without restarting it.
```

File: docs/configuration.md [hunk 3]
```diff
@@ -167,3 +167,29 @@ spec:
...
+
+PyMS has a feature to reload the configuration:
+
+```
+curl -X POST http://localhost:5000/reload-config
+```
+
+This endpoint call to the method `Microservice.reload_conf()`. This method restart the services,
```
Comment (alexppg) [inline docs/configuration.md:181]:
```suggestion
This endpoint calls the method `Microservice.reload_conf()`, which restarts the services, 
```

File: docs/configuration.md [hunk 4]
```diff
@@ -167,3 +167,29 @@ spec:
...
+In production environment, you could need to change configuration files without restart the microservice/pod/container.
+
+PyMS has a feature to reload the configuration:
+
+```
+curl -X POST http://localhost:5000/reload-config
+```
+
+This endpoint call to the method `Microservice.reload_conf()`. This method restart the services, 
+cript configuration and initialize `create_app`.
```
Comment (alexppg) [inline docs/configuration.md:182]: Is `cript` a typo?

File: docs/configuration.md [hunk 5]
```diff
@@ -167,3 +167,29 @@ spec:
...
+    def reload_conf(self):
+        self.delete_services()
+        self.config.reload()
+        self.services = []
+        self.init_services()
+        self.crypt.config.reload()
+        self.create_app()
+```
+
+It means that your libraries will be restarted, for this reason, it's important to initialize your BD,
```
Comment (alexppg) [inline docs/configuration.md:194]:
```suggestion
This means that your libraries will be restarted, which is why it's important to initialize your BD, 
```

**Review by alexppg — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 4 above)
Comment (avara1986) [reply docs/configuration.md]: yes... :see_no_evil:

File: docs/configuration.md [hunk 6]
```diff
@@ -167,3 +167,29 @@ spec:
...
+In a production environment you may need to change the microservice's configuration without restarting it.
+
+PyMS has a feature to reload the configuration:
+
+```
+curl -X POST http://localhost:5000/reload-config
+```
+
+This endpoint calls the method `Microservice.reload_conf()`, which restarts the services, 
+crypt configuration and initialize `create_app`.
```
Comment (alexppg) [inline docs/configuration.md:182]:
```suggestion
the encryption configuration and initializes `create_app`.
```

**Review by alexppg — CHANGES_REQUESTED** [review]

**Review by alexppg — APPROVED** [review]

**Pull request merged**